tests/functional/amdModuleCircular.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+	<head lang="en">
+	<meta charset="UTF-8">
+	<title>AMD Loading Test</title>
+</head>
+
+<body>
+	<script src="../../loader.js"></script>
```

Comment:
I am ok with not indenting the code only if the script tags are also not indented.  If you want to indent the script tags, then the code has to be indented as well.